docs/overview/definitions.md

```diff
@@ -35,6 +35,29 @@ Commandline Application for interacting with services on the MQ Bus.
 More information is available 
 [on Github](https://neongeckocom.github.io/neon-docs/neon_core/index).
 
+## Operations Documentation
+The Operations Documentation and other documents discussing code organization or
+processes use some specific terms to refer to different kinds of modules.
+
+### Plugin
+Refers to a module that may be replaced by an equivalent module. These will usually 
+extend a class from [ovos-plugin-manager](https://github.com/openvoiceos/ovos-plugin-manager).
+
+### Skill
+Refers to a class that extends `MycroftSkill`. A `skill` is a subset of a `plugin`.
+For the purposes of the operations documentation, a skill is distinctly different
+from a plugin.
+
+### Service
+Refers to a module that runs as a standalone service. This often includes a Docker
+container that runs the service. Many plugins may be run as a standalone service,
+but for the purposes of the operations documentation those are considered plugins
+and not services.
```

Comment:
I think I know what you mean here - PHAL plugins are not services in most cases, although they could be run standalone, but things like neon-audio or neon-speech are considered services. Correct?

Reply:
I debated listing examples but I wasn't sure if that would help for anyone unfamiliar with the project since it wouldn't be complete. My concept was anything that runs as a k8s service except TTS plugins, etc. which are Plugins first even though they publish containers and run standalone with a TTS Server wrapper

Reply:
Maybe worth listing those exceptions, but if you don't think it's helpful to new users not much point in adding it